src/app/_services/api.service.ts

```diff
@@ -3,6 +3,7 @@ import { HttpClient, HttpParams } from '@angular/common/http';
 
 import { Property } from '_models/property.model';
 import { Landlord } from '_models/landlord.model';
+// import { HousingAvailable } from '_models/housing-available.model';
```

Comment:
were you unable to get the model to work in the api service, or unsure if/how you should? on your httpClient call in your getHousingAvailables method, you can replace the cast response from <any> to <HousingAvailable[]> and it should cast the api response as an array of HousingAvailable models.